yourSolution.js

```diff
@@ -1,7 +1,22 @@
 // Name
 // Problem Description
 
-const solution = () => {
+const solution = (arr) => {
+  let obj = {};
+  for(let i = 0; i < arr.length; i++){
+    if(!obj[arr[i]]){
+      obj[arr[i]] = 1;
+    }
+    else{
+      obj[arr[i]]++;
+    }
+  }
+  for(let key in obj){
+    if(obj[key] > 1){
+      return true;
+    }
+  }
+  return false;
 };
 
```

Comment:
@ColinX13 yourSolution file should never be committed. It should only be used to solve solutions locally!

happy coding :)